spring-integration-jms/src/main/java/org/springframework/integration/jms/JmsOutboundGateway.java

```diff
@@ -1178,6 +1180,12 @@ public void onMessage(javax.jms.Message message) {
 			LinkedBlockingQueue<javax.jms.Message> queue = this.replies.get(correlationId);
 			if (queue == null) {
 				if (this.correlationKey != null) {
+					Log debugLogger = LogFactory.getLog("si.jmsgateway.debug");
```

Comment:
???
Can't we rely on the normal supplied `logger` from the super `IntegrationObjectSupport` ?